services/apiService.js

```diff
@@ -0,0 +1,23 @@
+const getAllTeamMembers = async (url) => {
+  const response = await fetch(url);
+  const data = await response.json();
+
+  const teamMembers = Object.values(data);
+  const teamMemberIds = Object.keys(data);
```

Comment:
Instead of splitting `teamMembers` and `teamMembersIds` you could use a pattern similar to `Object.entries(data).map(([key, value])=>({})` to create an array of objects that contain the data we want without separating the ids from teamMembers